src/components/popUp/index.js

```diff
@@ -0,0 +1,41 @@
+import React, {useState, useEffect } from 'react';
+import Styles from './styles.module.css';
+import IconButton from '../iconButton';
+import Button from '../button'
+import SignIn from '../signInForm'
+import SignUp from '../SignUp'
+
+const Popup = ({ isOpen, onClose, isNewUser, setNewUser}) => {
```

Comment:
Why do you need to pop up the arguments isNewUser and setNewUser?
sign up and login is the only place that using this component?
for making this comp more generic maybe add to the props {content}
Then its use will be much more generic